CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+1.8.2 Release notes (2025-02-05)
```

Comment:
wyrzućmy to stąd i daj do osobnego pliku w katalogu changelogs - tutaj potem sie to samo wstawi przy deploymencie

i 2025-03-05 :)